packages/minicaml/minicaml.0.3.2/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+maintainer: "sudo-woodo3@protonmail.com"
+authors: ["Alessandro Cheli"]
+homepage: "https://github.com/0x0f0f0f/minicaml"
+bug-reports: "https://github.com/0x0f0f0f/minicaml/issues"
+dev-repo: "git+https://github.com/0x0f0f0f/minicaml.git"
+license: "MIT"
+synopsis: "A simple, didactical, purely functional programming language"
+description: "A simple, didactical, purely functional programming language written for the programming 2 course at the University of Pisa, extended with a simple parser made with Menhir and ocamllex"
+doc: "https://github.com/0x0f0f0f/minicaml"
+
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  [make "test" "-j" jobs] {with-test}
+]
+depends: [
+    "dune" {>= "2.0"}
+    "ocaml" {>= "4.05.0"}
+    "ANSITerminal"
+    "menhir"
+    "ppx_deriving"
+    "cmdliner"
+    "alcotest" {with-test & >= "0.8.5"}
+    "bisect_ppx" {>= "1.4.1"}
```

Comment:
```suggestion
    "bisect_ppx" {with-test & >= "1.4.1"}
```
I don't understand, why is this not a test dependency anymore? `--conditional` makes it a conditional dependency (so depending on whether or not the package is installed, the ppx is going to be used or not)